funnel/models/event.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+from . import db, BaseMixin
+from .space import ProposalSpace
+from .user import User
+import random
+import uuid
+from sqlalchemy.ext.associationproxy import association_proxy
+
+__all__ = ['Event', 'TicketType', 'Participant', 'Attendee', 'SyncTicket']
+
+PRINTABLE_ASCII = map(chr, range(32, 127))
+ATTENDEE_TABLE_NAME = 'attendee'
+EVENT_TICKET_TYPE_TABLE_NAME = 'event_ticket_type'
+
+
+def make_key():
+    key = str(uuid.uuid4()).replace('-', '')
+    return ''.join(random.sample(key, len(key)))
+
+
+def make_public_key():
+    return make_key()[:8]
+
+
+def make_private_key():
+    return make_key()[:8]
+
+
+event_ticket_type = db.Table(EVENT_TICKET_TYPE_TABLE_NAME, db.Model.metadata,
+    db.Column('event_id', db.Integer, db.ForeignKey('event.id')),
```

Comment:
Or just add a `created_at` DateTime column with `default=datetime.utcnow`. You don't need `updated_at` since there's nothing to update.